docker/Dockerfile

```diff
@@ -52,6 +47,48 @@ RUN apt-get update \
         mixlib-log \
         mixlib-shellout \
         systemu \
+        # Gems for Mongo.
+        fluent-plugin-mongo \
+        mongo \
+    # Remove unused gem versions.
+    && /opt/google-fluentd/embedded/bin/gem uninstall httpclient -v 2.7.2 \
+    && rm -rf \
+        # Cache.
+        /var/cache \
+        # apt.
+        /usr/bin/apt-* \
+        /var/lib/apt/lists/* \
+        # dpkg.
+        /usr/bin/dpkg* \
+        /var/lib/dpkg \
+        # Temp files.
+        /tmp/* \
+        # LDAP.
+        /usr/lib/x86_64-linux-gnu/libldap* \
+        # IBM mainframe / EBCDIC specific encodings.
```

Comment:
Can we have the same order in the comment as in the actual gconv directories below?

Reply:
Done.

Reply:
I was hoping you'd reorder the comment, keeping the directories alphabetical, but this is fine too.